staff/almu-itur/js-arrays/push.js

```diff
@@ -0,0 +1,9 @@
+function push(arr, elem) {
+
+    if (arr == undefined) throw Error ('array is not valid');
+    if (elem == undefined) throw Error ('element not defined');
+
+    length = arr.length;
+    arr[length]=elem;
+    return arr;
```

Comment:
The push method returns the new array length. More info https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push

Reply:
You are right, this has been corrected now. Thank you!